data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,166 @@ date: 2021-05-27T21:07:30-00:00
 
 # Get Up and Running With OCaml
 
-This page will walk you through the installation of OCaml and the Dune build system, as well as offer support for
-your favourite text editor or IDE. These instructions work on Unix-based systems like Linux and macOS, and there are also instructions on how to install OCaml on Windows.
+This page will walk you through the installation of everything you need to have a comfortable development environment to write OCaml code on a new project. This includes of course [installing the compiler](#installing-ocaml) itself, but also a build system, a package manager, an LSP server to support your editor, and a few other tools that we describe [later](#setting-up-development-tools), setting up [editor support](#configuring-your-editor) and bootstrapping a [new project](#starting-a-new-project).
+
+If you are willing to set up OCaml on Windows, you might be interested in
+reading [OCaml on Windows](/docs/ocaml-on-windows) first.
+The following instructions work on Linux, BSD and macOS but also on Cygwin and
+WSL.
 
 ## Installing OCaml
 
-It's a straightfoward process to install OCaml. [Opam](https://opam.ocaml.org/) is OCaml's package manager, so we'll install it first. You will also use Opam when installing third-party OCaml libraries.
+The **platform installer** is currently in active development. It will
+automatically install Opam, OCaml and the development tools. [See below](#up-and-running-with-the-platform-installer)
+for the instruction.
+
+Please note that the installer is work in progress and might not work on your system.
+
+OCaml is available as a package in most linux distributions, however, it is
+often outdated. On the contrary, OCaml's package manager `opam` allows to
+easily switch from one version of OCaml to the other and much more. This is
+especially useful since different projects might require different versions of
+OCaml.
+
+So the best way to install `ocaml` is in fact by using `opam`, OCaml's package manager.
```

Comment:
```suggestion
OCaml is available as a package in most Linux distributions; however, it is
often outdated. On the contrary, OCaml's package manager opam allows you to
easily switch between OCaml versions and much more. This is
especially useful since different projects might require different versions of
OCaml.

So the best way to install OCaml is in fact by using opam, OCaml's official package manager.
```

I added "official" in the last sentence to emphasise the preference to use opam. Otherwise, it sounds repetitive since we just said it was OCaml's package manager above. If you don't like the "official," I suggest changing this sentence and combining it with the last to read: 

"This is especially useful since different projects might require different versions of OCaml, so the best way to install OCaml is in fact by using opam."